frame/nfts/src/lib.rs

```diff
@@ -271,13 +281,26 @@ pub mod pallet {
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config<I>, I: 'static = ()> {
 		/// A `collection` was created.
-		Created { collection: T::CollectionId, creator: T::AccountId, owner: T::AccountId },
+		Created {
+			collection: T::CollectionId,
+			creator: T::AccountId,
+			owner: T::AccountId,
+		},
 		/// A `collection` was force-created.
-		ForceCreated { collection: T::CollectionId, owner: T::AccountId },
+		ForceCreated {
+			collection: T::CollectionId,
+			owner: T::AccountId,
+		},
 		/// A `collection` was destroyed.
-		Destroyed { collection: T::CollectionId },
+		Destroyed {
+			collection: T::CollectionId,
+		},
 		/// An `item` was issued.
-		Issued { collection: T::CollectionId, item: T::ItemId, owner: T::AccountId },
+		Issued {
+			collection: T::CollectionId,
```

Comment:
I have run `rustup update`, not sure why this is formatted like this, but the fmt CI check is passing.

Reply:
try to run `cargo +nightly fmt`, it could be because you need to run it using the `nighty` channel

Reply:
I am aware of that, I always run the `fmt` command like that.